src/db/commit.rs

```diff
@@ -105,6 +105,18 @@ impl Commit {
         self.commit().value_hash().unwrap()
     }
 
+    pub fn next_mutation_id(&self) -> u64 {
+        let meta = self.commit().meta().unwrap();
```

Comment:
Not something that came in with this PR but: the highly overloaded nature of "commit" in this code I found super confusing. commit()'s signature is basically the following, which is hard to make heads or tails of:
```
struct Foo {}

impl Foo {
  fn foo() -> foo::Foo
}
```
Can we bring `super::commit_generated::commit` in as `commit_fb` or similar and call `commit()` something like `fb()` so the reader can see what is what?

Reply:
That's a good idea. https://github.com/rocicorp/repc/issues/100